src/settings.ts

```diff
@@ -0,0 +1,81 @@
+import * as vscode from "vscode";
+import * as os from 'os';
+import * as path from 'path';
+import * as fs from 'fs';
+import { State } from "./extension";
+import { Project, Group, Filter, generateSvgUri } from './utils';
+
+const settingFilename: string = path.join(os.homedir(), 'vscode_log_analysis.json');
+
+export function openSettings(state: State) {
```

Comment:
state is not used

Reply:
I fixed it.

Reply:
Same comment as above on not passing state var around.

Reply:
I fixed it.